core/controllers/cron.py

```diff
@@ -303,6 +304,23 @@ def get(self):
             )
 
 
+class CronAppFeedbackReportsScrubberHandler(base.BaseHandler):
+    """Handler for scrubbing app feedback reports that are expiring."""
+
+    URL_PATH_ARGS_SCHEMAS = {}
+    HANDLER_ARGS_SCHEMAS = {'GET': {}}
+
+    @acl_decorators.can_perform_cron_tasks
+    def get(self):
```

Comment:
I found this handler is for the corn job so not sure whether we can have post/delete requests! Ignore this comment in case it's not possible to make a post/delete request through cron job!

Reply:
From what I can tell all our cron handlers use `get` so I think we should leave as is.